docs/README.md

```diff
@@ -23,9 +23,17 @@ Hugo is available for many platforms. It can be installed using:
 Once installed, you may run the following from the `docs/site` directory
 to access a rendered view of the documentation:
 
-```bash
-hugo server --disableFastRender
-```
+1. Run
 
-Access the site at [http://localhost:1313](http://localhost:1313). Press
+    ```sh
+    make preview-build
+    ```
+
+1. Run
+
+    ```bash
+    hugo server --disableFastRender
+    ```
+
+Access the site at [http://localhost:1313](http://localhost:1313/docs/edge/). Press
```

Comment:
Let's update the displayed URL to be the full path too so it's more obvious that this isn't just going to the root of the local server in case they type the URL in instead of clicking in the README.